svir/thread_worker/upload_worker.py

```diff
@@ -84,12 +84,36 @@ def work(self):
         file_size_mb = file_size_mb / 1024 / 1024
         self.upload_size_msg = tr('Uploading ~%s MB...' % file_size_mb)
         self.set_message.emit(self.upload_size_msg)
-        permissions = {"authenticated": "_none",
-                       "anonymous": "_none",
-                       "users": [[self.username, "layer_readwrite"],
-                                 [self.username, "layer_admin"]]
+        permissions = {
+                       "admin": {
+                                 self.username: [
+                                     'change_layer_data',
+                                     'change_layer_style', 'add_layer',
+                                     'change_layer', 'delete_layer',
+                                     'view_resourcebase',
+                                     'download_resourcebase',
+                                     'change_resourcebase_metadata',
+                                     'change_resourcebase',
+                                     'delete_resourcebase',
+                                     'change_resourcebase_permissions',
+                                     'publish_resourcebase',
+                                  ]
+                                 },
+                       "users": {
+                                 self.username: [
+                                     'change_layer_data',
+                                     'change_layer_style', 'add_layer',
+                                     'change_layer', 'delete_layer',
+                                     'view_resourcebase',
+                                     'download_resourcebase',
+                                     'change_resourcebase_metadata',
+                                     'change_resourcebase',
+                                     'delete_resourcebase',
+                                     'change_resourcebase_permissions',
+                                     'publish_resourcebase',
+                                  ]
```

Comment:
In case you actually need the same list of permissions in both cases, I suggest to define the common list before defining `permissions`, and then to use it, therefore avoiding a dozen redundant lines of code.